roles/ui/files/FWO.UI/Program.cs

```diff
@@ -37,6 +37,7 @@
 
 builder.Services.AddScoped<AuthenticationStateProvider, AuthStateProvider>();
 builder.Services.AddScoped<CircuitHandler, CircuitHandlerService>();
+builder.Services.AddScoped<KeyboardInputService, KeyboardInputService>();
```

Comment:
could this have any performance impacts (as it is being used in the whole UI but only needed in  one component?